tests/unit/modules/test_netconfig.py

```diff
@@ -0,0 +1,117 @@
+"""
+    :codeauthor: Gareth J. Greenaway <gareth@saltstack.com>
+"""
+# Import Python libs
+
+import salt.modules.napalm_network as net_mod
+
+# Import Salt Libs
+import salt.states.netconfig as netconfig
+import salt.utils.files
+
+# Import Salt Testing Libs
+from tests.support.mixins import LoaderModuleMockMixin
+from tests.support.mock import MagicMock, patch
+from tests.support.unit import TestCase
+
+
+class NetconfigTestCase(TestCase, LoaderModuleMockMixin):
+    """
+    Test cases for salt.states.netconfig
+    """
+
+    def setup_loader_modules(self):
+        mock_true = MagicMock(return_value=True)
```

Comment:
Left over?

Reply:
I don't follow what you mean here. I wasn't familiar enough with the old testing system to actually write most of these myself. If they need to be migrated over to pytest in some way now I'll do it I just need to know what parts need to change.

Reply:
Sorry, should have expanded.
`mock_true = MagicMock(return_value=True)` is not being used, that's why I asked if it was a left over while coding this.

And don't worry about PyTest, it would be nice to have these written for PyTest, but it's not a requirement to accept and merge PRs :)

Reply:
Oh the left over was @garethgreenaway's fault :)

Reply:
ah I see. Well, he did write the entire thing for me as a favor so a big thank you to him anyway :) removed now